docs/about/roadmap.rst

```diff
@@ -0,0 +1,37 @@
+.. _about:
+
+Roadmap
+=======
+
+Overview
+--------
+
+As of autumn 2020 scipp is still in an "early preview" state.
+The overall plan for the coming releases is to:
+
+- Consolidate the existing generic functionality.
+- Add missing generic functionality to provide a more complex toolbox.
+- Weed out usability hurdles, to ensure the library's interface is easy to use.
+
+At this point we do not have a fixed release cycle.
+Instead we publish new releases when major feature additions or breaking changes have gone into scipp.
+
+Upcoming milestones
+-------------------
+
+v0.5 (expected end 2020 / early 2021)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This release focusses on:
+
+- Conceptual cleanup of ``Dataset`` and ``DataArray`` coordinates and masks, removing some ambiguities and inconsistencies.
+- A complete replacement of the mechanism used for "event data" with a new, more generic and flexible binning mechanism.
+
```

Comment:
Add something about the cleanup of plotting code and new features such as log buttons on the axes and profile picking and comparison tool?

Reply:
👍